mesheryctl/internal/cli/root/system/reset_test.go

```diff
@@ -0,0 +1,87 @@
+package system
+
+import (
+	"os"
+	"path/filepath"
+	"runtime"
+	"testing"
+
+	"github.com/layer5io/meshery/mesheryctl/pkg/utils"
+	log "github.com/sirupsen/logrus"
+)
+
+//var update1 = flag.Bool("update", true, "update golden files")
```

Comment:
because this has been already defined in other files so no sense in defining here, but using it directly below to update the files.

Reply:
@Revolyssup let's shift this flag globally to mesheryctl's root/constants package in your next unit testing PR.

Reply:
Yeah sounds good.

Reply:
Yup go it @piyushsingariya . I have done the required changes. Please have a review😅